src/pages/about/agreements/PrivacyPage.tsx

```
@@ -0,0 +1,10 @@
+import DOMPurify from 'dompurify';
+
+import termsHtml from './privacy.html?raw';
+
+const PrivacyPage = () => {
+  const sanitizedHtml = DOMPurify.sanitize(termsHtml);
+  return <div dangerouslySetInnerHTML={{ __html: sanitizedHtml }} />;
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)